src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateClaimRespondentNotificationHandler.java

```diff
@@ -62,24 +81,81 @@ private CallbackResponse notifyRespondentSolicitorForClaimIssue(CallbackParams c
             ? caseData.getApplicantSolicitor1UserDetails().getEmail()
             : caseData.getRespondentSolicitor1EmailAddress();
 
+        sendNotificationToSolicitor(caseData, recipient);
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(caseData.toMap(objectMapper))
+            .build();
+    }
+
+    //    Notify Claim Multiparty:
+    //
+    //            Scenario         Case Progresses       Callback Notify First Sol     Callback Notify Second Sol
+    //            1v1                  Online                    Solicitor 1                      -
+    //
+    //            1v2                  Online                    Solicitor 1                      -
+    //            (Same Sol)
+    //
+    //            1v2                  Online                    Solicitor 1                 Solicitor 2
+    //            (Different Sol
+    //            - Non Divergent)
+    //
+    //            1v2                  Offline             Solicitor 1 || Solicitor 2             -
+    //            (Different Sol
+    //            - Divergent)
+    private CallbackResponse notifyARespondentSolicitorForClaimIssue(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        String recipient;
+
+        CaseEvent caseEvent = CaseEvent.valueOf(callbackParams.getRequest().getEventId());
+        switch (caseEvent) {
+            case NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_ISSUE:
+                recipient = caseData.getRespondentSolicitor1EmailAddress();
+                if (shouldEmailRespondent2Solicitor(caseData)) {
```

Comment:
This does not look right as we have a separate case for this

Reply:
In the scenario where the user opts to Notify only the second solicitor in the Notfy Claim Options List.
This catches the users selection and sets the recipient to "Solicitor B" after this is processed, there is logic in the ```State Flow Engine``` and Camunda Diagram is configured to hand the case offline. As only 1 Solicitor is notified - the case will stay online as long as both are selected.

In camunda, this will trigger the second event to notify the other respondent solicitor :)